lib/ansible/modules/network/nxos/nxos_pim.py

```diff
@@ -108,12 +120,20 @@ def get_commands(module, existing, proposed, candidate):
     proposed_commands = apply_key_map(PARAM_TO_COMMAND_KEYMAP, proposed)
 
     for key, value in proposed_commands.items():
-        if key == 'ip pim ssm range' and value == 'default':
-            # no cmd needs a value but the actual value does not matter
-            command = 'no ip pim ssm range none'
-            commands.append(command)
-        else:
-            command = '{0} {1}'.format(key, value)
+        command = ''
+        if key == 'ip pim ssm range':
+            if value == 'default':
+                # no cmd needs a value but the actual value does not matter
```

Comment:
Is this true for all platforms?

Reply:
Yes. I tested this against all of our regression platforms.